languoids/tree/sign1238/vill1244/ghar1240/md.ini

```diff
@@ -25,7 +25,7 @@ hhbib_lgcode =
 
 [endangerment]
 status = moribund
-source = SafarWebster
+source = Safar & Webster 2014:9
```

Comment:
@d97hah hey man, you need to pull every once in a while. This one is coming back for the second time.